ffi/diplomat/cpp/examples/casemapping/Makefile

```diff
@@ -0,0 +1,28 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+.DEFAULT_GOAL := test
+.PHONY: build test
+FORCE:
+
+ALL_HEADERS := $(wildcard ../../include/*.hpp) $(wildcard ../../../c/include/*.h)
+
+CXX?=g++
+
+$(ALL_HEADERS):
+
+../../../../../target/debug/libicu_capi_staticlib.a: FORCE
+	cargo build -p icu_capi_staticlib --features cpp_default,icu_capi/icu_casemap
```

Comment:
All `icu_capi` features should be redirected because this only works inside the workspace

Reply:
I don't get it? The feature is already on `icu_capi` and will stay on `icu_capi`, so it should already be on `icu_capi_staticlib` and `icu_capi_cdylib` as well.

Reply:
huh